fairlearn/reductions/_moments/utility_parity.py

```diff
@@ -13,6 +13,10 @@
 _MESSAGE_INVALID_BOUNDS = "Only one of difference_bound and ratio_bound can be used."
 _DEFAULT_DIFFERENCE_BOUND = 0.01
 
+_POSITIVE_RATE_PARITY_IMPOSSIBLE_ERROR_MESSAGE_TEMPLATE = "The dataset " \
```

Comment:
This doesn't quite seem part of this PR

Reply:
Needed to make a few adjustments to the tests to avoid too much duplication for BGL, and this was one of a few small things that surfaced.

Reply:
I guess the question is whether this should raise or just not do anything. It is at the moment (pre this PR) a bug since we end up with an empty list due to 0 rows that correspond to label=1 (or 0 for the other one), and then we try to put this in a DataFrame and do math with it.

Reply:
This is not the right way to do it. Can we get this out of the current PR and file an issue?

The fix needs to happen in `UtilityParity`, because this bug occurs whenever we have zero events. In that case, we should just have zero constraints (vacuous constraints).

Reply:
OK, sure.

Reply:
opened #543 and removed this from the PR